pkgs/tools/networking/smartdns/default.nix

```diff
@@ -0,0 +1,54 @@
+
+{ stdenv, fetchFromGitHub, openssl, fetchpatch }:
+
+stdenv.mkDerivation rec {
+  pname = "smartdns";
+  version = "29";
+  src = fetchFromGitHub {
+    owner = "pymumu";
+    repo = pname;
+    rev = "Release${version}";
+    sha256 = "06kgmahgma0ybhacqzj5sfazmhzs3p27a0sxik4xbbk3dyqn9ykj";
+  };
+
+  buildInputs = [ openssl ];
+
+  patches = [
+    (fetchpatch {
+      url =
+        "https://git.archlinux.org/svntogit/community.git/plain/trunk/systemd.patch?h=packages/smartdns";
+      sha256 = "0jxzsgxz6y1a4mvwk6d5carxv4nxsswpfs1a6jkplzy2gr0g8281";
+    })
+  ];
+
+  prePatch = "chmod -R +w ../systemd";
+
+  patchFlags = [ "-p1" "../systemd/smartdns.service" ];
+
+  postPatch =
+    "substituteInPlace ../systemd/smartdns.service --replace /usr/bin/smartdns $out/bin/smartdns";
+
+  sourceRoot = "source/src";
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm644 ../systemd/smartdns.service $out/etc/systemd/system/smartdns.service
```

Comment:
There actually seems to be an install script. Would using `./install -i --prefix $out` not work?

Reply:
Hmm, they seem to be creating `$prefix/usr/sbin` instead of `$prefix/sbin`:

https://github.com/pymumu/smartdns/blob/b31792ad9b3ef8c3bf87dd0f8bae0d0a5b5c9122/install#L80

Could you suggest upstream they use a simple `Makefile` akin to https://github.com/medusalix/xow/blob/a35977eaada7b6cf8191e47e9f6bf5473b9ee18a/Makefile

Reply:
Yeah, I will address it in the same issue. But now it doesn't work.